### PR TITLE
fix: log ignored selectors correctly on error

### PR DESCRIPTION
**Pull request by robin-drexler**
**What**:

default `getElementError` implementation always logged that script and style are being ignored even when the default has been overridden with the `defaultIgnore` option.

It now logs the correct selector.
Since the selector could be any css selector, we can't just print `<script />` etc. Instead we just print the selector as is.

**Checklist**:

- [ ] Documentation added to the
      [docs site](https://github.com/testing-library/testing-library-docs)
- [x] Tests
- [ ] TypeScript definitions updated
- [ ] Ready to be merged

### CODE REVIEW

File: src/__tests__/element-queries.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
 import {render, renderIntoDocument} from './helpers/test-utils'
 
+// set original config
+let originalConfig
+beforeAll(() => {
+  originalConfig = getConfig()
+})
+
+afterEach(() => {
+  configure(originalConfig)
```
Comment (robin-drexler) [inline src/__tests__/element-queries.js:11]: Ensure tests don't rely on previous tests changing the config in a certain way.

File: src/__tests__/element-queries.js [hunk 2]
```diff
@@ -1092,6 +1102,19 @@ test('the default value for `ignore` can be configured', () => {
   expect(noStyle[1].tagName).toBe('DIV')
 })
 
+test('the default value for `ignore` is used in errors', () => {
```
Comment (robin-drexler) [inline src/__tests__/element-queries.js:1105]: New test that proves that overriding `defaultIgnore` produces a new error message

File: src/__tests__/wait-for.js [hunk 3]
```diff
@@ -194,14 +194,14 @@ test('when a promise is returned, if that is not resolved within the timeout, th
   await sleep(5)
 
   expect((await waitForError).message).toMatchInlineSnapshot(`
-Timed out in waitFor.
-
-Ignored nodes: comments, <script />, <style />
```
Comment (eps1lon) [inline src/__tests__/wait-for.js:199]: Can we keep the original formatting? "comments, scripts, style" looks like they all refer to the same thing while "scripts, style" specifically refers to elements.

Comment (eps1lon) [reply src/__tests__/wait-for.js:199]: Should be fine. Alternative is another `.split().map().join()` which is probably too expensive.

**Review by eps1lon — APPROVED** [review]
Thanks!

**Comment by eps1lon** [comment]
@all-contributors add @robin-drexler for code

**Pull request merged**